server/src/Game/Physics/Wind.js

```diff
@@ -0,0 +1,16 @@
+import Physics from './_Physics';
+
+export class Wind extends Physics {
```

Comment:
Please use default export instead
```suggestion
export default class Wind extends Physics {
```